README.md

```diff
@@ -56,22 +55,21 @@ The terafoundation level configuration is as follows:
 | --------- | -------- | ------ | ------ |
 | brokers | List of seed brokers for the kafka environment | String[] | optional, defaults to `["localhost:9092"]` |
 | security_protocol | Protocol used to communicate with brokers, may be set to `plaintext` or `ssl` | String | optional, defaults to `plaintext` |
-| ssl_ca_location | File or directory path to CA certificate(s) for verifying the broker's key | String | only used when `security_protocol` is set to `ssl` |
+| ssl_ca_location | File or directory path to CA certificate(s) for verifying the broker's key. Ignored if `ssl_ca_pem` is provided. | String | only used when `security_protocol` is set to `ssl` |
+| ssl_ca_pem | CA certificate string (PEM format) for verifying the broker's key. If provided `ssl_ca_location` will be ignored. | String | only used when `security_protocol` is set to `ssl` |
```

Comment:
Please rename `ssl_ca_pem` to be `caCertificate` like it is with minio.

No need to change other setting for now.  Actually doing so would break existing jobs.